### PR TITLE
books section is operational

### PR DESCRIPTION
**Pull request by suyash-20**
I have added the functionality of "read from" and "write to" csv files for the book management project.

### CODE REVIEW

File: application1.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    print("________________________________________________\n")
+    
+    print("Here's our catalogue to explore around:")
+    print("""
+    1. Books
+    2.Authors
+    3.Book Reviews
+    """)
+
+    choice = int(input("Enter your choice: "))
```
Comment (ulhas) [inline application1.py:35]: What happens when the choice is not an integer?

File: application1.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+2. Add books to the catalogue.
+		""")
+		
+        user_input = int(input("Enter your choice as '1' or '2': "))
+        
+        if user_input == 1:
+            print("Here's our Book catalogue:")
+            
+            for row in book_names:
+                print("|\t",row[0],"|\t", row[1],"|\t", row[2],"|\t", row[3])
```
Comment (ulhas) [inline application1.py]: convert this into a Book object. Then add a method inside the Book object to print it.

Comment (ulhas) [reply application1.py]:
```book = Book(id=row[0]. ...)
book.print()```

File: models.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Book(object):
+    def __init__(self, id, name, author_id, year_of_publication):
+        self.id = id
+        self.name = name
+        self.author_id = author_id
+        self.year_of_publication = year_of_publication
```
Comment (ulhas) [inline models.py:7]: make a method in this class called `print()`

File: application1.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import csv
+
+book_file = open("books.csv",'rt')
+book_names = csv.reader(book_file)
```
Comment (ulhas) [inline application1.py]: You are getting `books` not book_names.

Comment (ulhas) [inline application1.py]: Make an Enum for choices. `MainChoice`. `BookChoice`

File: application1.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    print("Here's our catalogue to explore around:")
+    print("""
+    1. Books
+    2.Authors
+    3.Book Reviews
+    """)
+
+    choice = int(input("Enter your choice: "))
+
+    if choice ==1:
```
Comment (ulhas) [inline application1.py]: if `choice == MainChoice.Book`

**Review by ulhas — CHANGES_REQUESTED** [review]

**Comment by suyash-20** [comment]
Made all the necessary changes as per Ulhas's code review.

**Pull request closed without merging**